test/Microsoft.AspNet.Mvc.FunctionalTests/project.json

```diff
@@ -3,7 +3,7 @@
     "../Microsoft.AspNet.Mvc.Formatters.Xml.Test/XmlAssert.cs"
   ],
   "compilationOptions": {
-    "define": [ "__RemoveThisBitTo__GENERATE_BASELINES" ],
+    "define": [ "__RemoveThisBitTo__GENERATE_BASELINES", "FUNCTIONAL_TESTS" ],
```

Comment:
One of the types in TestCommon gets included here and it used to exclude Moq out using `#if MOQ_SUPPORT`. This replaces it.